electrum/gui/qml/components/CpfpBumpFeeDialog.qml

```diff
@@ -56,113 +56,99 @@ ElDialog {
-                        snapMode: Slider.SnapOnRelease
-                        stepSize: 1
-                        from: 0
-                        to: cpfpfeebumper.sliderSteps
-                        onValueChanged: {
-                            if (activeFocus)
-                                cpfpfeebumper.sliderPos = value
-                        }
-                        Component.onCompleted: {
-                            value = cpfpfeebumper.sliderPos
-                        }
-                        Connections {
-                            target: cpfpfeebumper
-                            function onSliderPosChanged() {
-                                feeslider.value = cpfpfeebumper.sliderPos
-                            }
+                    Layout.fillWidth: true
+
+                    GridLayout {
+                        width: parent.width
+                        columns: 2
+
+                        Label {
+                            Layout.preferredWidth: 1
+                            Layout.fillWidth: true
+                            text: qsTr('Total size')
+                            color: Material.accentColor
                         }
-                    }
 
-                    FeeMethodComboBox {
-                        id: feemethod
-                        feeslider: cpfpfeebumper
-                    }
-                }
+                        Label {
+                            Layout.preferredWidth: 1
+                            Layout.fillWidth: true
+                            text: cpfpfeebumper.totalSize + ' ' + UI_UNIT_NAME.TXSIZE_VBYTES
+                        }
 
-                Label {
-                    visible: feemethod.currentValue
-                    text: qsTr('Target')
-                    color: Material.accentColor
-                }
+                        Label {
+                            text: qsTr('Input amount')
+                            color: Material.accentColor
+                        }
 
-                Label {
-                    visible: feemethod.currentValue
-                    text: cpfpfeebumper.target
-                }
+                        FormattedAmount {
+                            amount: cpfpfeebumper.inputAmount
+                        }
 
-                Label {
-                    text: qsTr('Fee for child')
-                    color: Material.accentColor
-                }
+                        Label {
+                            text: qsTr('Output amount')
+                            color: Material.accentColor
+                        }
 
-                FormattedAmount {
-                    amount: cpfpfeebumper.feeForChild
-                    valid: cpfpfeebumper.valid
-                }
+                        FormattedAmount {
+                            amount: cpfpfeebumper.outputAmount
+                            valid: cpfpfeebumper.valid
+                        }
```

Comment:
Let's just remove both the `Input amount` and the `Output amount` labels/amounts. They have become redundant with the inputs/outputs inspector.